webextensions/javascript-apis.json

```diff
@@ -3453,7 +3453,7 @@
                     "version_added": "48.0"
                   },
                   "opera": {
-                    "version_added": false
+                    "version_added": true
```

Comment:
> BTW, how about removing InjectDetails here and from the MDN wiki? 

I think it's probably a good idea. They used to be the same, and macros in `executeScript` and `insertCSS` included them from `InjectDetails`, but it got awkward, so we ended up duplicating it. I don't think we should remove the MDN page in case links to it exist, but we could stop linking to it and update it to refer people to `executeScript` and `insertCSS`.